garage/torch/data_management/simple_replay_buffer.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+from garage.torch.data_management.replay_buffer import ReplayBuffer
+
+
+class SimpleReplayBuffer(ReplayBuffer):
```

Comment:
I think we can just use the ReplayBuffer we have.

Reply:
They have very different APIs. Seems like syncing them up requires more than trivial work. For now, let's focus on divergent APIs exposed to end users. 

That being said we definitely need to have them unified in the long term. Could you give shot here and estimate how much effort we might need? @CatherineSue

Reply:
I've taken a look at the replay_buffers here. Though APIs are different, the utility of each function looks like the same. So I think a couple of hours is sufficient to do this. Definitely less than the time that would take on unifying samplers. 

I could help on this task since I'm familiar with ReplayBuffer in garage. Looks like this would belong to the 3rd step and 4th step in your head comments.

Reply:
Let's just get a demo of torch running before we spend time unifying interfaces. It's not clear if we're going to merge this into garage directly, or just keep it as a branch to make sure we can support torch once we split the repo.